frontend/src/lib/components/SmoothingFilter/smoothingFilterLogic.ts

```diff
@@ -16,7 +17,9 @@ export const smoothingFilterLogic = kea<smoothingFilterLogicType>({
     }),
     listeners: ({ actions, props }) => ({
         setSmoothing: async ({ filter }) => {
-            insightLogic(props).actions.setFilters({ smoothing_intervals: filter })
+            insightLogic(props).actions.setFilters({
```

Comment:
This should actually call `actions` which is connected above rather than another instance of insightLogic
```suggestion
            actions.setFilters({
```

Reply:
But it wont work until we refactor the IntervalLogic